.gitignore

```diff
@@ -1,7 +1,6 @@
 secrets.yml
 users.yml
 extra_vars.yml
-*.txt
```

Comment:
@Spredzy why are you removing *.txt here?

Reply:
@IPvSean because a file named `test-requirements.txt` is being added so `yamllint` can be installed whenn contributing (And allowing `tox` to be run)

Reply:
Updated to only not ignore `!test-requirements.txt`